diff --git a/index.d.ts b/index.d.ts
index 53dc74d77277c7fa4fd459caa7ed261bf3a0ea02..632f9c358baff836bc3e6ad585e4a4861e843487 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -19,6 +19,7 @@
 import * as bodyParser from 'body-parser';
 import * as serveStatic from 'serve-static';
 import * as core from 'express-serve-static-core';
+export { Express } from 'express-serve-static-core';
 import * as qs from 'qs';
 
 /**